File: AIO/AIO 2021/[p2] Art Class II.py
rawinput = open('artin.txt', 'r').readlines()
N = int(rawinput[0])
H = rawinput[1:]
 
maxx = 0
minx = 100001
maxy = 0
miny = 100001
 
for i in H:
    x, y = map(int, i.split())
    if x > maxx:
        maxx = x
    if x < minx:
        minx = x
    if y > maxy:
        maxy = y
    if y < miny:
        miny = y
 
output_file = open('artout.txt', 'w')
output_file.write(str((maxx-minx)*(maxy-miny)))
output_file.close()

File: AIO/AIO 2021/[p2] Art Class II.cpp
#include <bits/stdc++.h>
using namespace std;
 
int main() noexcept {
    static int N, X, Y;
    cin >> N;
 
    static int maxX = 0, minX = 100001, maxY = 1, minY = 100001;
 
    for(static int i = 0; i < N; i++) {
        cin >> X >> Y;
        maxX = max(maxX, X);
        maxY = max(maxY, Y);
        minX = min(minX, X);
        minY = min(minY, Y);
    }
 
    cout << (maxX - minX)*(maxY - minY);
}
